release-notes/release-notes59.md

```diff
@@ -367,9 +367,9 @@ For information on:
 
 ## Security fixes
 
-{{site.productname}} 5.9 provides fixes for the following security issues:
+{{site.productname}} 5.9 provides fixes for the premium pre-configured Dockerfiles for containerizing the server-side components, also known as the Docker starter kits. These Dockerfiles been updated to address various security issues.
```

Comment:
This isn't correct to my knowledge, the WAR files are the parts that had the dependencies updated, not the docker files (though they do pull in the WAR files).

Reply:
Either way, these security updates apply to all server side items, not just the docker files.

Reply:
The dockerfiles were updated as well. They now use jetty's image directly, rather than dealing with gpg keyservers themselves.

Reply:
So the dockerfile updates were more about making them work more often (rather than need patching every time jetty changed contributors)

Reply:
Updated the notes in commit https://github.com/tinymce/tinymce-docs/pull/2099/commits/a1b0741dd32a722366d5cb4f433953f210e1bbdd